lit/tests/SmokeTest.sc

```diff
@@ -37,12 +37,12 @@ class FooModule extends Module {
 
 args.head match {
   case "chirrtl" => {
-    print(circt.stage.ChiselStage.emitCHIRRTL(new FooModule))
+    println(circt.stage.ChiselStage.emitCHIRRTL(new FooModule))
   }
   case "panama-firrtl" => {
-    print(lit.utility.panamaconverter.firrtlString(new FooModule))
+    println(lit.utility.panamaconverter.firrtlString(new FooModule))
   }
   case "panama-verilog" => {
-    print(lit.utility.panamaconverter.verilogString(new FooModule))
+    println(lit.utility.panamaconverter.verilogString(new FooModule))
   }
```

Comment:
why changing this?

Reply:
The output may not contain an ending newline character, which may cause matching problems if more output is added after it. Changing `print` to `println` is harmless but safer.